docs/vyos.vyos.vyos_firewall_global_module.rst

```diff
@@ -354,7 +354,7 @@ Parameters
                 <td>
                 </td>
                 <td>
-                        <div>Allows you to specify a brief description for the port group.</div>
+                        <div>A  brief description for the port group.</div>
```

Comment:
2 whitespaces after `A`

Reply:
These are autogenerated from the `DOCUMENTATION` in the modules, please focus there. It looks like you've also commented on these.